sdk/identity/identity/src/msal/browserFlows/msalAuthCode.ts

```diff
@@ -118,10 +118,15 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
   /**
    * Uses MSAL to trigger a redirect or a popup login.
    */
-  public async login(scopes: string | string[] = []): Promise<AuthenticationRecord | undefined> {
+  public async login(
+    scopes: string | string[] = [],
+    loginOptions: MsalBrowserLoginOptions
+  ): Promise<AuthenticationRecord | undefined> {
     const arrayScopes = Array.isArray(scopes) ? scopes : [scopes];
     const loginRequest = {
-      scopes: arrayScopes
+      scopes: arrayScopes,
+      state: loginOptions.state,
```

Comment:
Are state and nonce the only values that are actually supported? Where are the other values passed as part of loginOptions used?

Reply:
Thank you for pointing this out!